### PR TITLE
importMIDN, importSHEN, importNCRN, importERMN directory fix

### PR DESCRIPTION
**Pull request by KateMMiller**
removed some of the initial directory code and pasted Dir into file directory for the network import files.

### CODE REVIEW

**Comment by johnpauls** [comment]
Looks good, but I will test in office on  Monday before merging.

File: R/importNCRN.R [hunk 1]
```diff
@@ -11,25 +11,23 @@
...
+  InSaps<-read.csv(paste(Dir,"Saplings.csv",sep="/"),as.is=T, header=T)
+  InSeeds<-read.csv(paste(Dir,"Seedlings.csv",sep="/"),as.is=T, header=T)
+  InShrubs<-read.csv(paste(Dir,"Shrubs.csv",sep="/"),as.is=T, header=T)
+  InShSeeds<-read.csv(paste(Dir,"Shrub_Seedlings.csv",sep="/"),as.is=T, header=T)
+  InVines<-read.csv(paste(Dir,"Vines.csv",sep="/"),as.is=T, header=T)
+  InHerbs<-read.csv(paste(Dir,"Herbs.csv",sep="/"),as.is=T, header=T)
+  InCommons<-read.csv(paste(Dir,"CommonNames.csv",sep="/"), as.is=T, header=T)
   InCommons$Common[InCommons$NCRN_Common!=""]<-InCommons$NCRN_Common[InCommons$NCRN_Common!=""]
   InCommons$TSN<-as.character(InCommons$TSN)
   setwd(OldDir)
```
Comment (johnpauls) [inline R/importNCRN.R]: This line should be deleted

File: R/importERMN.R [hunk 2]
```diff
@@ -12,21 +12,19 @@
...
-  InSaps<-read.csv("Saplings.csv",as.is=T, header=T)
-  InSeeds<-read.csv("Seedlings.csv",as.is=T, header=T)
-  InCommons<-read.csv("CommonNames.csv", as.is=T, header=T)
+  InTrees<-read.csv(paste(Dir,"Trees.csv",sep="/"),as.is=T, header=T)
+  InSaps<-read.csv(paste(Dir,"Saplings.csv",sep="/"),as.is=T, header=T)
+  InSeeds<-read.csv(paste(Dir,"Seedlings.csv",sep="/"),as.is=T, header=T)
+  InCommons<-read.csv(paste(Dir,"CommonNames.csv",sep="/"), as.is=T, header=T)
   InCommons$Common[InCommons$NCRN_Common!=""]<-InCommons$NCRN_Common[InCommons$NCRN_Common!=""] ##maybe change this to ERMN?
   InCommons$TSN<-as.character(InCommons$TSN)
   setwd(OldDir)
```
Comment (johnpauls) [inline R/importERMN.R]: This line should be deleted

File: R/importMIDN.R [hunk 3]
```diff
@@ -10,25 +10,23 @@
...
   #InGrowthRates<-read.csv("treecalc.csv",as.is=T, header=T)
-  InSeeds<-read.csv("Seedlings.csv",as.is=T, header=T)#  each row is individual seedling with height and browse
+  InSeeds<-read.csv(paste(Dir,"Seedlings.csv",sep="/"),as.is=T, header=T)#  each row is individual seedling with height and browse
   #InSeedsQuad<-read.csv("Seedlings_Quad.csv",as.is=T, header=T)# quadrat level summary of tree seedlings
-  InVines<-read.csv("Vines.csv",as.is=T, header=T)
-  InCommons<-read.csv("CommonNames.csv", as.is=T, header=T)
+#  InVines<-read.csv("Vines.csv",as.is=T, header=T)
+  InCommons<-read.csv(paste(Dir,"CommonNames.csv",sep="/"), as.is=T, header=T)
   InCommons$TSN<-as.character(InCommons$TSN)
   setwd(OldDir)
```
Comment (johnpauls) [inline R/importMIDN.R]: This line should be deleted

File: R/importSHEN.R [hunk 4]
```diff
@@ -12,24 +12,22 @@
...
+  InTrees<-read.csv(paste(Dir,"Trees.csv",sep="/"),as.is=T, header=T)
+  InSaps<-read.csv(paste(Dir,"Saplings.csv",sep="/"),as.is=T, header=T)
+  InSeeds<-read.csv(paste(Dir, "Seedlings.csv",sep="/"),as.is=T, header=T)
+  InShrubs<-read.csv(paste(Dir,"Shrubs.csv",sep="/"),as.is=T, header=T)
+  InShSeeds<-read.csv(paste(Dir,"Shrub_Seedlings.csv",sep="/"),as.is=T, header=T)
+  InHerbs<-read.csv(paste(Dir,"Herbs.csv",sep="/"),as.is=T, header=T)
+  InCommons<-read.csv(paste(Dir,"CommonNames.csv",sep="/"), as.is=T, header=T)
   InCommons$Common[InCommons$NCRN_Common!=""]<-InCommons$NCRN_Common[InCommons$NCRN_Common!=""]
   InCommons$TSN<-as.character(InCommons$TSN)
   setwd(OldDir)
```
Comment (johnpauls) [inline R/importSHEN.R]: This line should be deleted

File: R/importNCRN.R (hunk 1 above)
Comment (KateMMiller) [reply R/importNCRN.R]: Snap! Sorry for not catching that. For some reason I thought that line was at the very end of the function, so missed that it was still there. I went through and deleted them, and pushed the files.

File: R/importMIDN.R [hunk 5]
```diff
@@ -10,28 +10,25 @@
...
-  InSaps<-read.csv("Saplings.csv",as.is=T, header=T)
+  InTrees<-read.csv(paste(Dir,"Trees.csv", sep="/"),as.is=T, header=T)
+  InSaps<-read.csv(paste(Dir,"Saplings.csv",sep="/"),as.is=T, header=T)
   #InGrowthRates<-read.csv("treecalc.csv",as.is=T, header=T)
-  InSeeds<-read.csv("Seedlings.csv",as.is=T, header=T)#  each row is individual seedling with height and browse
+  InSeeds<-read.csv(paste(Dir,"Seedlings.csv",sep="/"),as.is=T, header=T)#  each row is individual seedling with height and browse
   #InSeedsQuad<-read.csv("Seedlings_Quad.csv",as.is=T, header=T)# quadrat level summary of tree seedlings
-  InVines<-read.csv("Vines.csv",as.is=T, header=T)
-  InCommons<-read.csv("CommonNames.csv", as.is=T, header=T)
+#  InVines<-read.csv("Vines.csv",as.is=T, header=T)
```
Comment (KateMMiller) [inline R/importMIDN.R]: The # before line 28 should be deleted. MIDN has vine data that can be imported to match what NCRN has. Sorry I had that wrong at first.

**Comment by KateMMiller** [comment]
Looks good to me, an ran fine when I tested it.

**Pull request merged**